tests/NovelRT.Tests/Interop/Maths/QuadTreePointTest.cpp

```diff
@@ -28,15 +28,13 @@ TEST(InteropQuadTreePointTest, deleteReturnsSuccess)
 
 TEST(InteropQuadTreePointTest, deleteReturnsNullInstanceFailureWhenGivenNullptr)
 {
-    const char* outputError = nullptr;
```

Comment:
It appears in some cases these output errors have commented out comparison checks with TODOs on them.

My instinct says deleting the output error in these cases might not be ideal. If you're planning on fixing tests here, then consider actually doing the fix - otherwise just comment the unused bits out if the compiler is whining.

Reply:
Doh, didn't catch the commented line below, sorry.

Reply:
`Nrt_getErrMsgIsNullptr` is not an existing function. Since I'm not sure if it's something that has been removed or is just missing I'll just comment the declaration of outputError.
I'm all for fixing this but if the function is to be implemented I'd need to know what it's supposed to do.

Reply:
It got renamed, I forget it's new name. It'll be inside one of our interop utility headers.